src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,40 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_LENGTH = 6;
```

Comment:
The name of the constant `MINIMUM_LENGTH` is not descriptive enough to specify whether it refers to the minimum length of the login or password. Consider renaming it to `MINIMUM_PASSWORD_LENGTH` or `MINIMUM_LOGIN_LENGTH` accordingly.